dt-impact-tracer/src/main/java/edu/washington/cs/dt/impact/plugin1/Sample.java

```diff
@@ -0,0 +1,628 @@
+package edu.washington.cs.dt.impact.plugin1;
+
+import com.reedoei.testrunner.mavenplugin.TestPlugin;
+import com.reedoei.testrunner.mavenplugin.TestPluginPlugin;
+import com.reedoei.testrunner.testobjects.TestLocator;
+import edu.washington.cs.dt.impact.Main.InstrumentationMain;
+import edu.washington.cs.dt.impact.runner.OneConfigurationRunner;
+import edu.washington.cs.dt.impact.runner.Runner;
+import edu.washington.cs.dt.impact.tools.OutputPrecomputedDependences;
+import edu.washington.cs.dt.impact.tools.detectors.FailingTestDetector;
+import edu.washington.cs.dt.main.ImpactMain;
+import edu.washington.cs.dt.tools.UnitTestFinder;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.text.StrSubstitutor;
+import org.apache.maven.cli.MavenCli;
+import org.apache.maven.project.MavenProject;
+import org.codehaus.plexus.classworlds.realm.ClassRealm;
+import scala.collection.JavaConverters;
+
+import java.io.*;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Sample extends TestPlugin {
+    // Constants For runPrecomputedDependencies
+    static final int MEDIANTIMES = 5;
```

Comment:
Minor: Access modifiers/final on all fields, unless it really should be package-private.